examples/builder-example/app.js

```diff
@@ -3,7 +3,7 @@ const { builder } = require('../../build');
 const staticOptions = { diamorphosis: { configFolder: './examples/simple-example' } };
 
 const w = builder(staticOptions)
-  .forTypescript()
+  .forTypescript(false)
```

Comment:
why?

Reply:
Because the example is in js not in ts. 
But you get how you could use it.
Feel free to add ts examples!